RiotSwiftUI/Modules/Room/PollHistory/View/PollListItem.swift

```diff
@@ -0,0 +1,76 @@
+// 
+// Copyright 2023 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct PollListData {
+    let startDate: Date
+    let question: String
+}
+
+struct PollListItem: View {
+    @Environment(\.theme) private var theme
+    
+    private let pollData: PollListData
+    
+    init(pollData: PollListData) {
+        self.pollData = pollData
+    }
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 12) {
+            Text(pollData.formattedDate)
+                .foregroundColor(theme.colors.tertiaryContent)
+                .font(theme.fonts.caption1)
+
+            HStack(alignment: .firstTextBaseline, spacing: 8) {
+                Image(uiImage: Asset.Images.pollHistory.image)
+                    .resizable()
+                    .frame(width: 16, height: 16)
```

Comment:
Fixing a frame with a constant like this might not work as desired for users with dynamic type. Might be desirable to add an `@ScaledMetric` property on the view otherwise it will look like this:
![Screenshot 2023-01-13 at 2 23 59 pm](https://user-images.githubusercontent.com/6060466/212343261-9afb28f7-a98a-40f3-8f26-733909d60bac.png)